HW03/aleksei_zhuchkov/aleksei_zhuchkov_hw_03_t_02.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(data_log = 'error')
+    result = []
+    raise ArgumentError if data_log == 'error'
```

Comment:
we don't need it
but you can check that you receive a log, not a number or smth else

Reply:
I corrected my code and found another way. Can you check it?